src/edit_python_pe/main.py

```
@@ -1,420 +1,10 @@
-            name,
-            email,
-        )
-        self.exit(message=message)
-
-    async def on_event(self, event: Event) -> None:
-        # catch listview selection
-        if isinstance(event, ListView.Selected):
-            self.on_list_view_selected(event)
-
-        await super().on_event(event)
+from .app import MemberApp
```

Ruff (0.15.13)

[warning] The file is executable but no shebang is present

(EXE002)